browser/ui/views/tabs/brave_tab_context_menu_contents.cc

```diff
@@ -246,3 +267,66 @@ bool BraveTabContextMenuContents::IsValidContextMenu() const {
 void BraveTabContextMenuContents::OnMenuClosed() {
   menu_closed_ = true;
 }
+
+void BraveTabContextMenuContents::NewSplitView() {
+  auto* model = browser_->tab_strip_model();
+  auto* split_view_data = SplitViewBrowserData::FromBrowser(browser_);
+  CHECK(!split_view_data->IsTabTiled(model->GetTabHandleAt(tab_index_)));
+  chrome::AddTabAt(browser_, GURL("chrome://newtab"), tab_index_ + 1,
+                   /*foreground*/ true);
+  split_view_data->TileTabs(
+      std::make_pair(model->GetTabHandleAt(tab_index_),
+                     model->GetTabHandleAt(tab_index_ + 1)));
+}
+
+void BraveTabContextMenuContents::CloseSplitView() {
+  auto* model = browser_->tab_strip_model();
```

Comment:
Similar - the CloseSplitViewForActiveTab is pretty similar. Do we know the right-clicked tab in the `TabStripModel`?